plugins/org.jkiss.dbeaver.core/src/org/jkiss/dbeaver/ui/preferences/PrefPageDatabaseUserInterface.java

```diff
@@ -258,7 +257,18 @@ private void setSettings() {
                 clientTimezone.setText(TimezoneRegistry.getGMTString(timezone));
             }
         }
-        enableScreenReaderSupport.setSelection(store.getBoolean(DatabaseEditorPreferences.PREF_SCREEN_READER_ACCESSIBILITY));
+        DatabaseEditorPreferences.screenReaders.forEach(c -> cmbScreenReaderSupport.add(c));
```

Comment:
If you want to use functional APIs, use their full power.

```suggestion
        DatabaseEditorPreferences.screenReaders.forEach(cmbScreenReaderSupport::add);
```

Reply:
Used